pkg/plugins/trivy/plugin.go

```diff
@@ -41,6 +41,7 @@ const (
 )
 
 const (
+	GCPCR_Inage_Regex         = `^(gcr\.io.*|^([a-zA-Z0-9-]+)-*-*.docker.pkg.dev.*)`
```

Comment:
is this regex with suffix `docker.pkg.dev.*` will cover all gcr use case or is it specific ?

Reply:
please feel free to do any changes to you want. normally it should cover gcr or AR like europe-docker.pkg.dev

Reply:
@chen-keinan please feel free to add anything.